src/cartesian/Line.js

```diff
@@ -336,9 +339,12 @@ class Line extends Component {
                 }
 
                 // magic number of faking previous x and y location
-                const interpolatorX = interpolateNumber(width * 2, entry.x);
-                const interpolatorY = interpolateNumber(height / 2, entry.y);
-                return { ...entry, x: interpolatorX(t), y: interpolatorY(t) };
+                if (that.animateNewValues) {
```

Comment:
There is no need to read `animateNewValues` from `that`, just read this from `this.props`.